src/Centreon/Domain/PlatformTopology/PlatformTopologyService.php

```diff
@@ -479,4 +486,18 @@ private function findParentPlatformAndSetId(PlatformTopology $platformTopology):
         }
         return null;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function findLocalhostMonitoringName(): ?string
```

Comment:
Do not create a method only to retrieve its name.
Return an entity and then retrieve its name